server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/DomainsRoutesTest.java

```diff
@@ -95,30 +94,24 @@ void setUp() throws Exception {
 
         @Test
         void getDomainsShouldBeEmptyByDefault() {
-            List<String> domains =
-                given()
-                    .get()
-                .then()
-                    .statusCode(HttpStatus.OK_200)
-                    .contentType(ContentType.JSON)
-                    .extract()
-                    .body()
-                    .jsonPath()
-                    .getList(".");
-
-            assertThat(domains).isEmpty();
+            given()
+                .get()
+            .then()
+                .statusCode(HttpStatus.OK_200)
+                .contentType(ContentType.JSON)
+                .body(".", hasSize(0));
```

Comment:
We should discuss this: I personally prefer extract + assertj has we don't have to learn a second assertion library and obviously matcher objects are way inferiors to builder pattern (it's the reason why we stopped using junit assertion for assertj in the first place).
Furthermore, I profiled our test execution and rest-assured is a slow library. I once started to remove it from our code base. I don't say we should do that now but relying more on it doesn't seem the right thing to do.

Reply:
I never really knew much the difference between the two to be honest. I just thought this way is less verbose. But if you tell me you checked and this way is slower than extract and assertj then I will pay more attention to follow this pattern next time. Thanks for the feedback